x-pack/platform/plugins/shared/fleet/server/routes/cloud_connector/handlers.ts

```diff
@@ -76,3 +83,91 @@ export const getCloudConnectorsHandler: FleetRequestHandler<
     });
   }
 };
+
+export const getCloudConnectorHandler: FleetRequestHandler<
+  { cloudConnectorId: string },
+  undefined
+> = async (context, request, response) => {
+  const fleetContext = await context.fleet;
+  const { internalSoClient } = fleetContext;
+  const cloudConnectorId = request.params.cloudConnectorId;
+  const logger = appContextService
+    .getLogger()
+    .get('CloudConnectorService getCloudConnectorHandler');
+
+  try {
+    logger.info(`Getting cloud connector ${cloudConnectorId}`);
+    const result = await cloudConnectorService.getById(internalSoClient, cloudConnectorId);
+    logger.info(`Successfully retrieved cloud connector ${cloudConnectorId}`);
+    return response.ok({ body: result });
```

Comment:
I would like to see `body` enforce a type, for all cloud connector handlers

you can see an example here:
https://github.com/elastic/kibana/blob/869f04a49c52a26610ce3ea17b0e5597622cb6bf/x-pack/platform/plugins/shared/fleet/server/routes/agent_policy/handlers.ts#L304

this means that new `*CloudConnectorResponse` types will need to be added to the `rest_spec` folder I mentioned before

please follow our existing convention of returning `{item: ...}` for returning single objects or `{items: ...}` for returning multiple objects (i.e. list endpoint). existing `*CloudConnectorRequestSchema` will need to be updated as well